### ISSUE TITLE
Error upon running the rake tasks

### ISSUE THREAD

**Issue by kevieman** [consul/consul#3021]
### Description
I'm trying to do a clean install of Consul on my local machine.

### Current Behavior
Upon executing the rake tasks I get an error.
```
$ bin/rake db:create

/usr/bin/env: 'ruby': No such file or directory
```
### Expected Behavior
[comment]: # (Describe what you expected to happen.)

### Steps to reproduce
Follow the installation guide in the documentation.

**My system information:**
OS: Ubuntu 18.04.1 LTS (Also tried with 16.04.05 LTS)
git version: 2.17.1
rvm: 1.29.4 (manual) 
Ruby: 2.3.2
bundler: (default: 1.16.6)
Node: v10.13.0
psql (PostgreSQL) 11.0 (Ubuntu 11.0-1.pgdg18.04+2)
ChromeDriver 2.37.544315

**Comment by javierm** [comment]
Hi @kevieman. I see you've also mentioned this issue in our tech questions channel. That's actually the right place to ask :+1:.

I'm closing this issue since we don't offer technical support through GitHub. Hope you don't mind!

**Comment by kevieman** [comment]
No problem @javierm. :+1: 

I solved it either way and just in case I will put my solution here.
In the pas when i tried to install Consul locally I couldn't execute the rake tasks at all. I needed to do:
```
sudo bin/rake db:create
 ```
So now when I tried that I got that error message. Without the sudo I still couldn't execute it. I got the following error:
```
psql `rescue in connect': FATAL: database "consul_development" does not exist
```
I solved this by executing just:
```
rake db:create
```
This solved it but as a result it didn't fill the database completely when seeding it.
